### DISCUSSION TITLE
Reset Search bar

### DISCUSSION THREAD

**Discussion by anishsarkar** [filamentphp/filament#959] [Help]
I am unable to find it in the documentation. Perhaps it is easily possible to have a cross type icon, which will clear the search bar.

In fact if there is also an reset any filter that would also be great.

**Comment by danharrin** [comment]
Hey, I think most browsers should add a cross to `<input type="search">` automatically - which browser are you using?

**Reply by anishsarkar** [reply]
Thanks for your reply. I use Firefox. It looks like Firefox does not implement it. Just a quick search in stack overflow seems to confirm it. 

https://stackoverflow.com/questions/37846023/input-type-search-no-longer-shows-cancel-button-x-under-firefox
